pkg/apis/cluster/v1alpha2/zz_generated.deepcopy.go

```diff
@@ -225,6 +225,42 @@ func (in *Machine) DeepCopyObject() runtime.Object {
 	return nil
 }
 
+// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
```

Comment:
Do we need a pull-related check to validate that these are up to date?

Reply:
Agreed, I'll open an issue

Reply:
Filed https://github.com/kubernetes-sigs/cluster-api/issues/1172